src/main/java/com/epam/aidial/core/storage/ResourceDescription.java

```diff
@@ -200,7 +202,16 @@ public static ResourceDescription fromPrivateUrl(String url, EncryptionService e
         return description;
     }
 
+    @SneakyThrows
     public static ResourceDescription fromAnyUrl(String url, EncryptionService encryption) {
+        if (url == null) {
```

Comment:
I would avoid such changes. this method used in a plenty places (where only dial urls acceptable)

Reply:
fixed